app/config/tstvhq/events.json

```diff
@@ -1,84 +1,56 @@
 {
     "events": [
         {
-            "name": "Dark Arias",
-            "description": "Darker arias and art songs sung by a Friends of TST Delaware member (MarissaYoung.com)",
-            "room": "teaRoom",
-            "start": "2020-08-20T20:30:00-04:00",
-            "end": "2020-08-20T21:30:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/events/front parlor movie.jpg"
-        },
-        {
-            "name": "Salem Witch Lecture",
-            "description": "Learn about the Salem Witch Trials from our own Salem Witch, Thomas Vallor (@SalemWitchWiles). This lecture will take place every 30 minutes or until he smokes too much weed.",
+            "name": "Salem Walking Tours",
+            "description": "Salem's history from a Satanic perspective by local Satanic Salem witch Thomas Vallor. This is an hour long presentation.",
             "room": "backParlor",
-            "start": "2020-08-20T19:00:00-04:00",
-            "end": "2020-08-20T01:00:00-04:00",
+            "start": "2020-10-10T02:00:00-04:00",
+            "end": "2020-10-10T03:00:00-04:00",
```

Comment:
All of these times are formatted incorrectly. The time offset (in this case, `-04:00`) indicates that the timestamp is in a particular time zone corresponding to that offset. So if the event is at 8PM EDT, you can just write `2020-10-09T22:00:00-04:00`.